Chainkey icns & Darkcoin addition

ELM4Ever opened the pull request.

Commit:
chain-specific URIs

Commit:
show full URI

Kefkius committed:
Merge pull request #1 from Kefkius/uris

Chain-specific URI support

Commit:
Merge branch 'master' into develop

Commit:
Chainkey icns

guruvan commented:
@museack could you pass on the newer icon to @ELM4Ever ?

museack commented:
![maza-85x85](https://cloud.githubusercontent.com/assets/7134287/6203963/4026f6b0-b4f7-11e4-97bd-b0eb47af7460.png)
This one?  Not sure how to send it on github, other than adding into this message!  I think this is the same icon guruvan uses as his avatar, too.

museack commented:
![encompass-logo](https://cloud.githubusercontent.com/assets/7134287/6204031/0d95272e-b4f9-11e4-8cfe-354b32b82182.jpg)

Commit:
Darkcoin addition

darkcoin_hash library has to be installed first (X11 Hashing)
Terminal into darkcoin_hash directory, then type “sudo python setup.py
install”

Commit:
Merge branch 'develop' of https://github.com/mazaclub/encompass

ELM4Ever commented:
Cool - do you need icns or ico files for it?  
BTW - just added Darkcoin to the chains.  We will have a few servers by the end of the week.  Already one running at the moment.

museack commented:
Its just a first draft of the logo as a concept to see how everyone liked it. I haven't tried to make icns or ico files for it, not sure how well it will scale down.

Kefkius commented:
Ahh, looks like you merged mazaclub:develop @ELM4Ever, it's kind of messed up at the moment... Do you need those commits in the develop branch or can you omit them for this?

ELM4Ever commented:
Sorry about that.  I'm still struggling with Github commands.... ignore the :develop merged files.

EDIT:  The Darkcoin addition consists only of mods to lib/chain/_init_.py + cryptocur.py + darkcoin.py + the addition of the darkcoin_hash library in lib/    (from https://github.com/vertoe/darkcoin_hash) This library could be useful for other coins using X11 hashing also - it was developed by the team at Darkcoin.

Commit:
Chainparam.py

Commit:
darkcoin added to setup.py

Commit:
removed build in lib/darkcoin_hash

The whole library can be cloned from
https://github.com/vertoe/darkcoin_hash instead

Commit:
added uDRK

vertoe commented:
@ELM4Ever can you work on master branch without the develop merge?
i never tried to undo a merge before but try `git revert 9c4b7927d6341dc0d81d509f4eb255a2677bf6e2`

ELM4Ever commented:
Didn't work, starting from scratch.
